tests/acceptance/tests/Accessibility/StorefrontAccessibility.spec.ts

```diff
@@ -18,7 +18,15 @@ test('The Storefront should implement accessibility best practices.', { tag: '@A
     StorefrontAccountProfile,
     StorefrontAccountAddresses,
     StorefrontAccountPayment,
+    InstanceMeta,
 }) => {
+
+    // eslint-disable-next-line
+    if (InstanceMeta.features['V6_7_0_0']) {
+        // eslint-disable-next-line playwright/no-skipped-test
+        test.skip(true, 'This test is incompatible with V6_7_0_0');
+    }
```

Comment:
linter rules exist to not be omitted :)

```suggestion
    test.skip(InstanceMeta.features['V6_7_0_0'], 'This test is incompatible with V6_7_0_0');
```